tests/jerry/es2015/forin-header-strict.js

```diff
@@ -27,3 +27,35 @@ try {
 } catch (e) {
   assert(e instanceof SyntaxError);
 }
+
+var reached = false;
+
+for (var i in {}) {
+  reached = true;
+}
+assert(!reached);
+
+for (var i of []) {
```

Comment:
The `reached` should be reseted to a `false` state, otherwise the first "true" will make all other assert-s pass.

Reply:
ahh.. I've misunderstood the check. Sorry.